src/GroupedEndpoints/GroupedEndpointsFactory.php

```diff
@@ -24,12 +28,12 @@ public static function fromApp(
         GenerateDocumentation $command,
         RouteMatcherInterface $routeMatcher,
         bool $preserveUserChanges,
-        string $docsName = 'scribe'
+        CacheConfiguration $docsName
```

Comment:
This seems like mismatching parameter names? Name `$docsName` and type `CacheConfiguration` (also in the other places in this file)
```